Automation/login.ps1

```diff
@@ -0,0 +1,14 @@
+param(
+        [string] [Parameter(Mandatory=$true)] $tenantId,
+        [string] [Parameter(Mandatory=$true)] $applicationId,
+        [string] [Parameter(Mandatory=$true)] $secret,
+        [string] [Parameter(Mandatory=$true)] $subscriptionId
+      )
+
+[securestring]$secureSecret = ConvertTo-SecureString $secret -AsPlainText -Force      
+[pscredential]$credObject = New-Object System.Management.Automation.PSCredential ($applicationId, $secureSecret)
+Connect-AzAccount -ServicePrincipal -Credential $credObject -Tenant $tenantId      
+write-output "Logged into the account $applicationId"
+
+Set-AzContext -Subscription $subscriptionId
+write-output "Set to subscription $subscriptionId"
```

Comment:
Is the subscriptionId required for context when logging in with a service principal?

Reply:
No, it was just to set the subscription/context, since I think it makes sense to do that at the same time as logging in.